Test collection fails

### What I'm trying to achieve
Running tests through pytest tests/ on master branch

### Steps to reproduce the problem
1. Run the command pytest/tests
2. Get the following error message
```
ImportError while importing test module '/home/egoncearov/dev/saleor/tests/test_checkout.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_checkout.py:28: in <module>
    (0, True, reverse('cart:index')),
venv/lib/python3.6/site-packages/django/urls/base.py:57: in reverse
    app_list = resolver.app_dict[ns]
venv/lib/python3.6/site-packages/django/urls/resolvers.py:480: in app_dict
    self._populate()
venv/lib/python3.6/site-packages/django/urls/resolvers.py:416: in _populate
    for url_pattern in reversed(self.url_patterns):
venv/lib/python3.6/site-packages/django/utils/functional.py:36: in __get__
    res = instance.__dict__[self.name] = self.func(instance)
venv/lib/python3.6/site-packages/django/urls/resolvers.py:536: in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
venv/lib/python3.6/site-packages/django/utils/functional.py:36: in __get__
    res = instance.__dict__[self.name] = self.func(instance)
venv/lib/python3.6/site-packages/django/urls/resolvers.py:529: in urlconf_module
    return import_module(self.urlconf_name)
venv/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
saleor/urls.py:17: in <module>
    from .graphql.api import schema
saleor/graphql/api.py:6: in <module>
    from .account.mutations import (
saleor/graphql/account/mutations.py:12: in <module>
    from ..core.types import Error
E   ImportError: cannot import name 'Error'
______________________________ ERROR collecting tests/api/test_account.py _______________________________
ImportError while importing test module '/home/egoncearov/dev/saleor/tests/api/test_account.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/api/test_account.py:13: in <module>
    from saleor.graphql.account.mutations import SetPassword
saleor/graphql/account/mutations.py:12: in <module>
    from ..core.types import Error
E   ImportError: cannot import name 'Error'
```

### What I expected to happen
Successfully collect the tests

### Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->

**System information**
Operating system: Ubuntu 18.04
Browser: N/A

--- a/saleor/graphql/account/mutations.py
+++ b/saleor/graphql/account/mutations.py
@@ -9,7 +9,7 @@ from graphql_jwt.decorators import permission_required
 from ...account import emails, models
 from ...core.permissions import MODELS_PERMISSIONS, get_permissions
 from ..core.mutations import BaseMutation, ModelDeleteMutation, ModelMutation
-from ..core.types import Error
+from ..core.types.common import Error
 from ..order.mutations.draft_orders import AddressInput
 from ..utils import get_node
 

